src/material-experimental/mdc-form-field/form-field.ts

```diff
@@ -301,6 +321,7 @@ export class MatFormField implements AfterViewInit, OnDestroy, AfterContentCheck
               private _defaults?: MatFormFieldDefaultOptions,
               @Optional() @Inject(MAT_LABEL_GLOBAL_OPTIONS) private _labelOptions?: LabelOptions,
               @Optional() @Inject(ANIMATION_MODULE_TYPE) public _animationMode?: string) {
+    this._adapter = new TextFieldAdapter(this);
```

Comment:
could we set up the adapter above when we declare the `_adapter` class member?

Reply:
@devversion Yep!